lang/ilias_en.lang

```diff
@@ -1440,6 +1440,8 @@ assessment#:#tst_failed#:#Failed
 assessment#:#tst_failed_imp_qst_skl_assign#:#The question's assignments to the following competences could not be created. The according competences could not be identified within the local system.
 assessment#:#tst_failed_imp_skl_thresholds#:#The Import of thresholds for the following competences were skipped, because the according competences within the local system are configured with a different list of levels.
 assessment#:#tst_feedback#:#Feedback
+assessment#:#tst_feedback_is_given_inline#:#The feedback is given in your answer.
```

Comment:
Better: "The feedback will be displayed along with your answer." Suggested by Chris Potter ;-)

Reply:
Changed the text as suggested.